package.json

```diff
@@ -7,14 +7,17 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "codecov:e2e": "codecov -f ./coverage/coverage-final.json",
-    "codecov:unit": "codecov -f ./coverage/jest/coverage-final.json",
-    "codecov": "codecov -f coverage/coverage-final.json && codecov -f coverage/jest/coverage-final.json",
+    "codecov:e2e": "codecov -f ./jest-coverage/coverage-final.json",
+    "codecov:unit": "codecov -f ./jest-coverage/coverage-final.json",
+    "codecov": "codecov -f jest-coverage/coverage-final.json && codecov -f cypress-coverage/coverage-final.json",
+    "copy:reports": "mkdir -p reports/ && cp cypress-coverage/coverage-final.json reports/from-cypress.json && cp jest-coverage/coverage-final.json reports/from-jest.json",
+    "combine:reports": "mkdir -p reports/ && npx nyc merge reports && mv coverage.json .nyc_output/out.json",
     "test:unit": "jest",
     "test:e2e": "npx cypress run",
-    "test": "npm run test:unit && npm run test:e2e"
+    "test": "mkdir -p jest-coverage/ && npm run test:unit > jest-coverage/coverage.txt && npm run test:e2e"
```

Comment:
@vickywane looking closer at the Cypress and Jest documentation, it appears like we should be uploading as github artifact a text or HTML version of the report produced by nyc merge, instead of trying to use separate txt files. See here: https://istanbul.js.org/docs/advanced/coverage-object-report/

Cypress seems to only be able to output json or a [brief txt summary](https://docs.cypress.io/guides/tooling/code-coverage#See-code-coverage-summary). The latter is not very useful for debugging.

Reply:
Based on recent resolutions, we are uploading the coverage report generated by `CodeCov` as an artifact for each CI run.

This will lead to the removal of the scripts which manually combine the reports. I am awaiting your confirmation in this [comment](https://github.com/ambianic/ambianic-ui/pull/562#issuecomment-886605719) before removing the scripts.